roles/platform/tasks/initialize_teardown.yml

```diff
@@ -14,6 +14,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: Refresh Environment Info with Descendants
+  cloudera.cloud.env_info:
```

Comment:
As discussed, this functionality will go away from the `env` module. However, you can replace it with a call to `cloudera.exe.info` and it will populate a `deployment` variable via `set_fact` that you can use in lines 27-30

Reply:
given the nature of that change, and that it's deprecating a feature that is already present (if undesirable), I would rather do that separately.